pkg/boltzrpc/boltzrpc.proto

```diff
@@ -918,3 +928,18 @@ message ChangeWalletPasswordRequest {
     string old = 1;
     string new = 2;
 }
+
+message GetSwapMnemonicRequest {}
+message GetSwapMnemonicResponse {
+    string mnemonic = 1;
+}
+
+message SetSwapMnemonicRequest {
+    oneof mnemonic {
+        string existing = 1;
+        bool generate = 2;
```

Comment:
Having it this explicit is fine. And `optional string mnemonic = 1;` would do the trick too though